app/front_page/content.html

```diff
@@ -3,23 +3,23 @@
 
   <div class="col-xs-3">
   <select class="form-control" ng-model="vm.transfer.type" ng-disabled="vm.transfer.components.length > 0">
-    <option value="standard" selected>Standard</option>
-    <option value="unzipped bag">Unzipped Bag</option>
-    <option value="zipped bag">Zipped bag</option>
-    <option value="dspace">DSpace</option>
-    <option value="disk image">Disk Image</option>
+    <option value="standard" selected>{{ "Standard" | translate }}</option>
```

Comment:
Because the directive generates a new scope with an extra wrapping HTML element, e.g. `<option><span ng-scope>Standard</span></option>`. I've found that to cause some issues with our CSS - rarely happening but I thought adopting filters would make the resulting HTML layout cleaner in general. I compared the performance and it was unnoticeable although I'm sure filters won't scale for really big templates.